File: 005-minimum-of-three/005-minimum-of-three.f
program main                                                                    
  implicit none                                                                 
  integer  :: a, b, c, n, ans(20)                                               
                                                                                
  open(10, file="input")                                                        
                                                                                
  do n = 1, 20                                                                  
     read(10,*), a                                                              
     read(10,*), b                                                              
     read(10,*), c                                                              
                                                                                
     if (a .lt. b .and. a .lt. c) then                                          
        print "(I9,$)", a                                                       
     elseif (b .lt. a .and. b .lt. c) then                                      
        print "(I9,$)", b                                                       
     else                                                                       
        print "(I9,$)", c                                                       
     endif                                                                      
  enddo                                                                         
                                                                                
  close(10)                                                                     
end program main

File: 005-minimum-of-three/005-minimum-of-three.m
for i= 1:length(A)
if A(i,1)<A(i,2)&&A(i,1)<A(i,3)
    C(i,1)=A(i,1);
elseif A(i,2)<A(i,1)&&A(i,2)<A(i,3)
    C(i,1)=A(i,2);
else
    C(i,1)=A(i,3);
end
i=i+1;
end